src/Microsoft.ML.TimeSeries/IidAnomalyDetectionBase.cs

```diff
@@ -12,100 +12,144 @@
 
 namespace Microsoft.ML.TimeSeriesProcessing
 {
-    /// <summary>
-    /// This transform computes the p-values and martingale scores for a supposedly i.i.d input sequence of floats. In other words, it assumes
-    /// the input sequence represents the raw anomaly score which might have been computed via another process.
-    /// </summary>
-    public abstract class IidAnomalyDetectionBase : SequentialAnomalyDetectionTransformBase<Single, IidAnomalyDetectionBase.State>
+    public class IidAnomalyDetectionBaseWrapper : IStatefulTransformer, ICanSaveModel
```

Comment:
>public [](start = 4, length = 6)

Could you an XML comment for everything that is public?

Reply:
There is plenty without XML (I would steel from the interface comments).

---
In reply to: [253746818](https://github.com/dotnet/machinelearning/pull/2344#discussion_r253746818) [](ancestors = 253746818)